evm/utils.js

```diff
@@ -1235,6 +1236,37 @@ async function relayTransaction(options, chain, contract, method, params, native
     );
 }
 
+async function getWeightedSigners(config, chain, options) {
+    printInfo(`Retrieving verifier addresses for ${chain.name} from Axelar network`);
+
+    let signers;
+
+    if (isValidAddress(options.keyID)) {
```

Comment:
why is this being duplicated? import from the amplifier gateway deployment script in verify-contract. Or use this in deployment as well